appcues/src/main/java/com/appcues/ui/AppcuesCompositions.kt

```diff
@@ -34,6 +35,8 @@ internal data class AppcuesPagination(val onPageChanged: (Int) -> Unit)
 
 internal val appcuesPaginationData = mutableStateOf(AppcuesPaginationData(1, 0, 0.0f))
 
+internal val LocalLogcues = staticCompositionLocalOf<Logcues?> { null }
```

Comment:
could we maybe change this to be similar to LocalContext. without using "?"

it seems like it does something like this:

```kotlin
val LocalContext = staticCompositionLocalOf<Context> {
    noLocalProvidedFor("LocalContext")
}

private fun noLocalProvidedFor(name: String): Nothing {
    error("CompositionLocal $name not present")
}
```

so basically we have to provide it or we get runtime exception, but since we will always provide it we basically avoid have to deal with ?.let scenarios

Reply:
yes! i was looking for a good way to do that thank you